tests/yam/agama/agama_install_with_cryptfs.pm

```diff
@@ -0,0 +1,32 @@
+## Copyright 2023 SUSE LLC
```

Comment:
please do not create another module here, we just need to pass the test name and perhaps some other subtle adjustment to `yam/agama/agama_installation.pm`

Reply:
Strongly disagree on that. `agama_installation.pm` untilizes a reboot after doing the installation, so this would lead into disaster because there is no code to handle encrpyted file systems. So it can't be done by just a subtle adjustment.

Reply:
oh, right, we did that a lot of time ago, it is quite messy to unify it, Could you please file a ticket for it? thx.